core/client/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -124,6 +126,9 @@ protected FileInStream(URIStatus status, InStreamOptions options) {
     mAlluxioStorageType = options.getAlluxioStorageType();
     mShouldCache = mAlluxioStorageType.isStore();
     mShouldCachePartiallyReadBlock = options.isCachePartiallyReadBlock();
+    if (mShouldCachePartiallyReadBlock) {
+      LOG.debug("CacheParitallyReadBlock is enabled.");
+    }
```

Comment:
If we have a debug message about the stream options, perhaps use the `options.toString` method to get all the info?